src/applications/personalization/profile/tests/components/direct-deposit/PaymentInformationBlocked.unit.spec.jsx

```diff
@@ -2,7 +2,7 @@ import React from 'react';
 import { shallow } from 'enzyme';
 import { expect } from 'chai';
 
-import PaymentInformationBlocked from '../../components/PaymentInformationBlocked';
+import PaymentInformationBlocked from 'applications/personalization/profile/components/direct-deposit/PaymentInformationBlocked';
```

Comment:
We _could_ set up a new Babel alias ([like we have for vet360](https://github.com/department-of-veterans-affairs/vets-website/blob/369b26b47357b6f049a1fd3abe59b9aa49a76b6f/.babelrc#L54)) so we could just do `import PaymentInformationBlocked from 'profile/components/direct-deposit/PaymentInformationBlocked'`. I'm a fan of that, but also understand the argument that that can lead to confusion.

Reply:
I like this approach a lot! Let me explore this in my follow up PR. Thanks for pointing this out!